apps/console-lite/src/app/hooks/use-local-values.spec.ts

```diff
@@ -3,7 +3,11 @@ import useLocalValues from './use-local-values';
 
 describe('local values hook', () => {
   it('state of wallet dialog should be properly handled', () => {
-    const { result } = renderHook(() => useLocalValues());
+    const { result } = renderHook(() =>
+      useLocalValues(() => {
+        /**/
```

Comment:
question (non-blocking): why is this comment needed here?

Reply:
I found ts validator complains there lack of parameter of type `() => void`. When I filled it up with `() => {}` eslinter started to complain with no-empty-functions rule. Finally don't know which suppressing is better :-)

Reply:
And using ts ignore here would also require an eslint ignore which is more annoying. Sounds reasonable, can we add a comment to that effect please?

Reply:
finally I added there different parameters so problem has been resolved :+1: